src/api/eidas/controller.ts

```diff
@@ -85,27 +84,26 @@ export default class Controller {
   }
 
   static async putEidasKeys(
-    did: string,
-    opts: EidasKeysData
+    eidasQecId: string,
+    opts: EidasKeysInput
   ): Promise<RedisInsertion> {
-    if (
-      !opts ||
-      !opts.p12 ||
-      !opts.keyType ||
-      opts.keyType !== constants.KeyTypes.RSA
```

Comment:
Not storing keyType anymore ? Not needed or stored elsewhere?

Reply:
The agreed open API specs does not contain key types. So, it is assuming always RSA. 
To pass the tests, I had to remove it. We can always discuss again to add it